packages/docs-reanimated/docs/layout-animations/layout-transitions.mdx

```diff
@@ -4,195 +4,376 @@ sidebar_position: 3
 
 # Layout transitions
 
-import DocsCompatibilityInfo from '../_shared/_docs_compatibility_info.mdx';
+Layout transition allows you to animate changes in the layout of elements.
 
-<DocsCompatibilityInfo />
+import LayoutTransitions from '@site/src/examples/LayoutTransitions';
+import LayoutTransitionsSrc from '!!raw-loader!@site/src/examples/LayoutTransitions';
+import ExampleVideo from '@site/src/components/ExampleVideo';
 
-The document explains how can you animate all layout changes for a specific view just by adding a single property to the view.
-To be precise how to animate positions and dimensions of components. What's important it will all happen entirely on UI thread without any communication through the bridge. There are plenty of ways in which you can animated layout changes however in contrast to entering and exiting animations they are not so regular. We've prepared a few predefined layout transitions, however if you want to create more custom and complex transition you can create your own.
+<InteractiveExample src={LayoutTransitionsSrc} component={LayoutTransitions} />
 
-## How to use predefined layout transition?
+## Predefined Transitions
+
+Below we listed all of the currently available predefined layout transitions with their [layout animation modifiers](/docs/fundamentals/glossary#layout-animation-modifier).
+
+## Linear transition
 
-### 1. Import chosen transition
+Linear transition animates both position and dimension in the same way.
+
+### Reference
 
 ```js
-// Transition is just an example and should be replaced by real animation. For Instance Layout
-import { Transition } from 'react-native-reanimated';
+import { LinearTransition } from 'react-native-reanimated';
+
+function App() {
+  return <Animated.View layout={LinearTransition} />;
+}
 ```
 
-### 2. Choose Animated Component which layout you want to animate
+### Modifiers
 
-```js
-    // AnimatedComponent - component created by createAnimatedComponent or imported from Reanimated
-    <AnimatedComponent layout={Transition} >
+#### Time-based <Optional/>
```

Comment:
We have also Time-based spring modifiers, Physics-based spring modifiers and common spring modifiers.
So I believe that we shouldn't use this term here.
```suggestion
#### Easing <Optional/>
```